src/codewhisperer/service/transformationResultsViewProvider.ts

```diff
@@ -315,9 +320,9 @@ export class ProposedTransformationExplorer {
                     codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
```

Comment:
Should move the getLogger() line to be inside the catch block

Reply:
For some reason it's not letting me comment right next to that line

Reply:
I agree it should be in the catch block. Will update